lib/clean-source-haskell.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs }: src:
```

Comment:
There should be a short comment header here what the use-case for this is. Also where it originally came from.